batch7crud-roll3/src/main/java/com/lftechnology/batch7crud/filter/ExceptionFilter.java

```diff
@@ -0,0 +1,47 @@
+package com.lftechnology.batch7crud.filter;
+
+import java.io.IOException;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+import javax.servlet.Filter;
+import javax.servlet.FilterChain;
+import javax.servlet.FilterConfig;
+import javax.servlet.ServletException;
+import javax.servlet.ServletRequest;
+import javax.servlet.ServletResponse;
+import javax.servlet.http.HttpServletResponse;
+
+import com.lftechnology.batch7crud.constant.CommonConstant;
+import com.lftechnology.batch7crud.controller.AuthenticationController;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'com.lftechnology.batch7crud.controller.AuthenticationController'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AUselessImportCheck)